registration/include/pcl/registration/transformation_estimation_lm.h

```diff
@@ -168,6 +168,19 @@ namespace pcl
           warp_point_ = warp_fcn;
         }
 
+        /** \brief Set regularization coefficients. Defaults to all 0s (no regularization).
+         *  \param[in] regularization_coeffs the regularization coefficients (tx, ty, tz, qx, qy, qz for rigid 6-DoF
+         *  transform)
+         */
+        void
+        setRegularizationCoefficients(const VectorX &regularization_coeffs)
+        {
+          assert(regularization_coeffs.rows() == warp_point_.nr_dims() &&
+              "Regularization coefficient vector must have same dimension as transformation");
+          PCL_WARN("Setting regularization_coeffs for %p\n", (void*)this);
```

Comment:
Please replace this assert with a check and PCL_WARN in case of failure.

Reply:
Done.